### ISSUE TITLE
Javascript in php file - Bracket colors acting up since 12.0.0

### ISSUE THREAD

**Issue by m-thorsen** [pouyakary/ProColors#13]
I noticed some inconsistency in Javascript bracket/parentheses/punctuation coloring within "<script>" tags in php-based files (Laravel Blade in my case).

It does not seem to affect .html or .js files.

New (12.*.0)
![kary-js-brackets](https://cloud.githubusercontent.com/assets/14907694/26298041/f33bdddc-3ed5-11e7-8988-55c91f4eda4c.jpg)

Old (11.1.0)
![kary-js-brackets-old](https://cloud.githubusercontent.com/assets/14907694/26298197/89827a3a-3ed6-11e7-95a9-0a5db9aaf833.jpg)

**Comment by pouyakary** [comment]
@m-thorsen  thanks a lot for the issue. I thought I've fixed it in v12 but it seems the problem is worse that I thought. Can you do me a favor and share some code snipped with file format? I'm not a PHP user so I have no Idea how to reproduce it

**Comment by m-thorsen** [comment]
No problem, glad to contribute as I love the theme ;)

To reproduce, you need to create a new file with language mode PHP (.php extension).
Add a script tag with a javascript functions, calls, arrays and/or objects. 
There should be no need for headers etc, just the script tag should be fine.

Here's an example (might not be good js, but it should cover most cases I can think of when it comes to the highlighting):

```
<script>
    function doWhatever(param) {
        if (param !== 'undefined') {
            var object = {
                'test-key' : [
                    'i-am-in-an-array',
                    'i-am-too'
                ],
                'key' : 'value'
            };
            return object;
        }
    }
    $(function(){
        $.post('http://example.com', doWhatever(), function(response) {
            console.log(response);
        });
    });
</script>
```

**Comment by pouyakary** [comment]
@m-thorsen. I just released `12.1.1` as a hotfix release to this issue. Hope that you'll never experience this problem again. The issue is closed for now but I if you ever experienced it again please reopen it.

Also thanks for the great snippet. Ita actually contained all the punctuations I needed to check so well done on choosing the code 👍 And thanks a lot for using the theme and reporting the problems you find. It helps a lot!
